contracts/src/interfaces/IPaymentVault.sol

```diff
@@ -14,21 +14,21 @@ interface IPaymentVault {
     /// @notice Emitted when a reservation is created or updated
     event ReservationUpdated(address indexed account, Reservation reservation);
     /// @notice Emitted when an on-demand payment is created or updated
-    event OnDemandPaymentUpdated(address indexed account, uint256 onDemandPayment, uint256 totalDeposit);
+    event OnDemandPaymentUpdated(address indexed account, uint128 onDemandPayment, uint128 totalDeposit);
     /// @notice Emitted when globalSymbolsPerBin is updated
-    event GlobalSymbolsPerBinUpdated(uint256 previousValue, uint256 newValue);
+    event GlobalSymbolsPerBinUpdated(uint128 previousValue, uint128 newValue);
     /// @notice Emitted when reservationBinInterval is updated
-    event ReservationBinIntervalUpdated(uint256 previousValue, uint256 newValue);
+    event ReservationBinIntervalUpdated(uint128 previousValue, uint128 newValue);
     /// @notice Emitted when globalRateBinInterval is updated
-    event GlobalRateBinIntervalUpdated(uint256 previousValue, uint256 newValue);
+    event GlobalRateBinIntervalUpdated(uint128 previousValue, uint128 newValue);
     /// @notice Emitted when priceParams are updated
     event PriceParamsUpdated(
-        uint256 previousMinNumSymbols, 
-        uint256 newMinNumSymbols, 
-        uint256 previousPricePerSymbol, 
-        uint256 newPricePerSymbol, 
-        uint256 previousPriceUpdateCooldown, 
-        uint256 newPriceUpdateCooldown
+        uint128 previousMinNumSymbols, 
+        uint128 newMinNumSymbols, 
+        uint128 previousPricePerSymbol, 
+        uint128 newPricePerSymbol, 
```

Comment:
How much gas savings are we talking here? I assume we're going to price this in mwei, but I wonder if we should just keep this in wei if the cost savings are trivial. 
(If we do need to use mwei, can we document that?)